COPYRIGHT

```diff
@@ -1,3 +1,4 @@
-Copyright 2008-2014, 2016, 2017, 2019, 2020 Association of Universities for Research in Astronomy, Inc. (AURA)
-Copyright 2015, 2017, 2018, 2019 The Trustees of Princeton University
-Copyright 2015, 2018, 2019 University of Washington
+Copyright 2008-2014 Association of Universities for Research in Astronomy, Inc. (AURA)
```

Comment:
Please add back the more modern dates for AURA. I think you went back to just the old LSST Corp dates.

I think it should be:

```suggestion
Copyright 2008-2014, 2016, 2017, 2019, 2020, 2022-2023 Association of Universities for Research in Astronomy, Inc. (AURA)
```

Reply:
Oh, I changed it locally and forgot to push

Reply:
Hmm, the `copyright.py` script reports an activity from AURA in 2018 as well. Here's one: https://github.com/lsst/pex_config/pull/28

Reply:
I decided not to include it because it was a trivial change that should not really result in asserting a copyright claim.

Reply:
Also, if there is a copyright.py script that people are using it needs to be fixed to understand that LSST Corp no longer owns copyright.

Reply:
This is what I see in the dev guide and it explicitly returns LSST Corporation, despite the docstring mentioning the transfer: https://github.com/lsst-dm/dm_dev_guide/blob/434dec6845dc2ae28eeb9f7c9b918617aff8eee1/legal/copyright.py#L115

Reply:
Okay. It's wrong.